src/coreclr/tools/aot/ILCompiler.MetadataTransform/Internal/Metadata/NativeFormat/Writer/NativeFormatWriterGen.cs

```diff
@@ -106,8 +101,8 @@ internal new ArraySignatureHandle Handle
 
         public MetadataRecord ElementType;
         public int Rank;
-        public Int32[] Sizes;
-        public Int32[] LowerBounds;
+        public int[] Sizes;
```

Comment:
Is there any reason to have this as PrimitiveType? I see a translator in [SchemaDef](https://github.com/dotnet/runtime/blob/main/src/coreclr/tools/Common/Internal/Metadata/NativeFormat/Generator/SchemaDef.cs#L194-L208) that is later on used in [WriterGen](https://github.com/dotnet/runtime/blob/main/src/coreclr/tools/Common/Internal/Metadata/NativeFormat/Generator/WriterGen.cs#L254) to autogenerate these changes